angular.json

```diff
@@ -29,6 +29,18 @@
             ],
             "scripts": [
               "node_modules/marked/lib/marked.js"
+            ],
+            "allowedCommonJsDependencies": [
```

Comment:
This is the list of CommonJS libraries that we or some of our libraries use. They prevent the tree-shakeability of the compilation process and we should therefore consider replacing them (see [How CommonJS is making your bundles larger](https://web.dev/commonjs-larger-bundles/))